config/samsung-gtelwifiue.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# DTB: From T560NUUEU1CRG1_T560NUBNN1CRG1_HOME.tar.md5
+
+OPTIONS=()
```

Comment:
Need to add the `V_LCD_3P3` as regulator here (`samsung,panel-extra-power-gpio1` downstream). Not sure about the other extra-power-gpios, CABC doesn't sound like a regulator...

Reply:
the CABC gpios go straight to the panel, apparently, whatever it's for

Reply:
CABC = Content Aware Brightness Control, I think it's like some mechanism for the panel to adjust the brightness depending on the content it is shown (think of very dark scenes in a movie maybe). I don't understand why it's GPIOs though (rather than some panel register setting). Also, since brightness goes through the RT8555, I don't see how the panel could even adjust the brightness itself.

Reply:
In any case, I'm rather sure it's not a regulator so it should not be defined as `--regulator ...` option. The V_LCD_3P3 however is a `regulator-fixed` that you should hook up to the panel in the device tree.

Reply:
ok, i think i did that properly